.github/workflows/cd.main.yml

```diff
@@ -0,0 +1,51 @@
+name: CD
+on:
+  push:
+    branches:
+      - main
+env:
+  ORGANIZATION_SLUG: templates
+  INSTRUQT_TOKEN: ${{ secrets.INSTRUQT_TOKEN }}
+jobs:
+  deploy-changes:
+    name: Deploy changes
+    runs-on: ubuntu-latest
+    steps:
+      - name: Checkout repository
+        uses: actions/checkout@v2
+      - name: Scoping tracks
+        run: |
+          #!/bin/bash
+
+          # create slugs array
+          slugs=()
+          # Read output into an array
+          shopt -s nullglob
+          dirs=(*/)
+          shopt -u nullglob
+
+          for track in ${dirs[@]}; do
```

Comment:
Still seems a bit complicated how we identify what dirs to push. Couldn't we simplify to something like this?

```bash
#!/bin/bash

for dir in */; do
        pushd $dir;
        # docker command track push here.
        popd;
done
```

Reply:
Absolutely. I had the intention of doing this in parallel, in case one of the tracks failed. But this isn't a problem anymore.

Much cleaner 🚀